core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -147,34 +220,29 @@ var LearnerDashboardPage = function() {
     expect(await feedbackMessage.first().getText()).toMatch(message);
   };
 
-  this.checkIncompleteExplorationSection = async function(explorationTitle) {
-    await this.navigateToInCompleteSection();
-    await this.navigateToIncompleteExplorationsSection();
-    await this.expectTitleOfExplorationSummaryTileToMatch(
-      explorationTitle);
-  };
+  this.navigateToCommunityLessonsAndCheckIncompleteExplorations = (
+    async function(
+        explorationTitle) {
+      await this.navigateToCommunityLessonsSection();
+      await this.expectTitleOfExplorationSummaryTileToMatch(explorationTitle);
+    });
```

Comment:
This indentation seems needlessly complex

Reply:
Done